web/headref_display.go

```diff
@@ -11,6 +11,10 @@ import (
 // through a websockets integration, the head ref will be able to see score updates submitted by other
 // refs in realtime.
 func (web *Web) headrefDisplayHandler(w http.ResponseWriter, r *http.Request) {
+	if !web.userIsExpected(w, r, []string{adminUser, refUser}) {
```

Comment:
should there be some sort of error message returned to the client here? something equivalent to `handleWebErr`